### DISCUSSION TITLE
Lookup on Project Task

### DISCUSSION THREAD

**Discussion by hminhduc** [burkybang/SuiteAdvanced-Feedback#73] [Q&A] [answered]
How can lookup filed id on Project Task. Why this page can not view any more info

**Comment by burkybang** [comment]
Can you provide a screenshot? I don't feel I have enough information to help answer your question.

**Reply by hminhduc** [reply]
On this screen, i can not use Ctr-I to lookup field id, other shortcut also do not show any more..
Why only one this Project Task is stop, other record type is not that..
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/6736560/215926290-5599a363-7752-4402-8327-5cb1c743f0aa.png">

**Reply by burkybang** [reply, marked as answer]
This page is made with what NetSuite calls the "new UI". There are various pages made with the new UI. SuiteAdvanced doesn't fully work on these newer pages because of how drastically different they are.

**Reply by hminhduc** [reply]
Thank you for reply, i hope this page also comming working in the near future!
